director/redirect.go

```diff
@@ -529,6 +545,7 @@ func RegisterDirector(ctx context.Context, router *gin.RouterGroup) {
 	// Establish the routes used for cache/origin redirection
 	router.GET("/api/v1.0/director/object/*any", RedirectToCache)
 	router.GET("/api/v1.0/director/origin/*any", RedirectToOrigin)
+	router.PUT("/api/v1.0/director/origin/*any", RedirectToOrigin)
```

Comment:
I'm uncertain if this should be a PUT request. From your description, this is retrieving the URL, not updating the director.

Reply:
That makes sense to me, let me see if I can just change the payload of the request and check that to see if we need this for a PUT or a GET (line 290). I know I was working on this part with Justin and I can't remember if there was a reason we made this PUT.